src/config/laracrud.php

```diff
@@ -94,6 +94,13 @@
             //'users'=>'App\Models\AdminUser::class'
         ]
     ],
+    'templates'=>[
+        /**
+         * Path to the templates folder
+         * @example 'views/vendor/YOUR/templates/' or __DIR__.'/../vendor/YOUR/PACKAGE/src/resources/templates/'
+         */
+        'path'=>null,
```

Comment:
This is is not required. When assets are published then users can modify files according to their need. Its critical because code are generated based on this files. So if a file is missing then error will generate. 
So its better to let user edit the templates file and then our system will generate code according to their modified template.

Reply:
I thought of a little more special occasion.
For example, you can specify a standard template in your company and provide it as a separate template package.
But I thought that you could sort this out by what you said.
This pull request is likely to be closed if it is not needed